flysystem/FieldMigration.php

```diff
@@ -0,0 +1,414 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\flysystem\Form\FieldMigration.
+ */
```

Comment:
Would it be worth us including the following?
- a link to the source of this patch
- license/credit to the author
- the purpose of the patch, and what it enables
- when it's safe to remove the patch (if ever)

Reply:
@spikeheap still playing, not sure this is the right way to do it at the moment, you can apply patches using composer so messing with that atm :)

Reply:
Now patching from Drupal website